api/tests/certification/course/integration/infrastructure/repositories/sco-certification-candidate-repository_test.js

```diff
@@ -112,32 +157,66 @@ describe('Certification | Course | Integration | Repository | SCOCertificationCa
         organizationId: anOrganizationId,
         division: '3ème A',
       }).id;
-      const thirdInAlphabeticOrderCandidateId = databaseBuilder.factory.buildCertificationCandidate({
+
+      // Candidates
+      const thirdInAlphabeticOrderCandidate = databaseBuilder.factory.buildCertificationCandidate({
         lastName: 'Zen',
         firstName: 'Bob',
         sessionId,
         organizationLearnerId: aOrganizationLearnerId,
-      }).id;
-      const firstInAlphabeticOrderCandidateId = databaseBuilder.factory.buildCertificationCandidate({
+      });
+      const firstInAlphabeticOrderCandidate = databaseBuilder.factory.buildCertificationCandidate({
         firstName: 'Smith',
         lastName: 'Aaron',
         sessionId,
         organizationLearnerId: yetAnotherOrganizationLearnerId,
```

Comment:
Pas lié à cette PR mais on a du aOrganizationLearnerId, anotherOrganizationLearner et un yetAnother... pas très facile à distinguer ces trois élèves 
Je propose à nouveau de mettre simplement des chiffres, plus simple :)
```suggestion
        organizationLearnerId: organizationLearnerId3,
```